lua/astrocommunity/editing-support/mini-operators/init.lua

```diff
@@ -0,0 +1,31 @@
+-- NOTE: this will override the default AstroNvim keybinding for opening neo-tree.
+--       the default prefix would be g, but this will also override some bindings.
+local prefix = "<Leader>o"
+
+local mappings = {
+  n = {
+    [prefix] = {
+      name = "Text edit operators",
+    },
+  },
+}
```

Comment:
1. I think overriding the neo-tree default is a no-go. @AstroNvim/astrocommunity-maintainers  Any ideas? 
2. Similar to other reviews: Lets use the same method for mappings as we do most of the other plugins.

Reply:
I agree not to override default neo-tree operator